### PR TITLE
Gripper overhaul + New borg sprites

### PR DESCRIPTION
**Pull request by NanakoAC**
Grippers basically torn up and remade, i went down to the root and incorporated them into cyborg click and inventory code. This should make them resistant against future bugs.
Fixes all current known gripper bugs. Grippers work properly with all the stuff they should. Minor tweaks to borg modules.

In addition, i went and raided VG, TG, paradise and bay repositories, and ported all their best sprites that were suitable for us. VG station was the most lucrative by far, about 70% of the new ones came from there.

Sadly i did not port paradise's mouse-in-a-hamsterball-on-a-chassis cyborg frame. but maybe in future
Pictures of new chassis:

Most are shared by a couple of modules, ive not repeated them below for simplicity. And i probably missed some

### CODE REVIEW

File: code/game/machinery/recharger.dm [hunk 1]
```diff
@@ -83,13 +74,6 @@ obj/machinery/recharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
 		G.loc = src
 		charging = G
 		update_icon()
-	else if(portable && istype(G, /obj/item/weapon/wrench))
```
Comment (skull132) [inline code/game/machinery/recharger.dm]: Where did this gooooo?

File: html/changelogs/Nanako-GripperChassis.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (skull132) [inline html/changelogs/Nanako-GripperChassis.yml]: You changed your name?

File: code/_helpers/items.dm [hunk 3]
```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```
Comment (Lohikar) [inline code/_helpers/items.dm]: There's a helper for this, defined here: https://github.com/Aurorastation/Aurora.3/blob/master/code/_helpers/unsorted.dm#L1209

Comment (NanakoAC) [inline code/_helpers/items.dm]: good to know, but my new function is a generic safety check.

The comment isnt really adequate, aside from preventing borgs dropping their modules (but still allowing the use of the contents of grippers) it also prevents hardsuit modules being dropped.

Its intended to be a central place to hook in any other relevant safety checks, and its run in any situation where stuff is getting dropped into things. Closets, crates, tables, containers, etc.

So, not just for borgs!

File: code/game/machinery/recharger.dm (hunk 1 above)
Comment (NanakoAC) [inline code/game/machinery/recharger.dm]: it was duplicate code, the same wrench interaction was stuck twice in the attackby proc, i just deleted the latter one. wrenching and unwrenching it still works fine

File: html/changelogs/Nanako-GripperChassis.yml (hunk 2 above)
Comment (NanakoAC) [inline html/changelogs/Nanako-GripperChassis.yml]: woops, thats the default on example.yml, ill fix it tomorrow

srsly going to bed now x

File: code/_helpers/items.dm (hunk 3 above)
Comment (skull132) [inline code/_helpers/items.dm]: https://puu.sh/tiZdI/7100b9b26e.png

Just uh. Delete the helper in unsorted.dm. It's not used anywhere.

**Comment by LordFowl** [comment]
Several of these sprites are nice, and several more are ridiculous. I maintain the idea that there should only be one combat borg sprite, to retain uniformity.

File: code/_helpers/items.dm (hunk 3 above)
Comment (NanakoAC) [inline code/_helpers/items.dm]: i think its good to have, has a purpose besides safety checks. i was sure lohikar did something with it. @Lohikar what was your use case aside from safety?

**Comment by NanakoAC** [comment]
updated, requested removals done. Spider sprites remade into several versions

File: code/_helpers/items.dm (hunk 3 above)
Comment (Lohikar) [inline code/_helpers/items.dm]: It _was_ designed as a safety check for engiborg tools, I thought I used it in welders (tools.dm), but apparently I just used an isrobot() call.

**Pull request merged**